apps/css-workshop/src/components/Tag.astro

```diff
@@ -0,0 +1,41 @@
+---
+const { title, className, id, style, button = true, variant, href, ...props } = Astro.props;
+---
+
+<>
+  {
+    variant === 'basic' && !href && (
+      <span class='iui-tag-basic' id={id}>
+        {title}
+      </span>
+    )
+  }
+  {
+    href && (
+      <a href={href} class='iui-anchor iui-tag-basic' id={id}>
+        {title}
+      </a>
+    )
+  }
+
+  {
+    variant !== 'basic' && (
+      <span className='iui-tag' id={id} style={style} {...props}>
+        <span class='iui-tag-label' title={title}>
+          {title}
+        </span>
+        {button && (
```

Comment:
This `button` prop is too vague. What kind of "button"? What does it do? Can I pass my own button into it?

Reply:
I'll make it more specific.